projects/demo-integrations/src/support/commands/paste.ts

```diff
@@ -8,12 +8,10 @@ export function paste<T extends Cypress.PrevSubjectMap['element']>(
     $subject: T,
     data: string,
 ): ReturnType<Cypress.CommandFn<'paste'>> {
-    const inputType = 'insertFromPaste';
     const element = Cypress.dom.unwrap($subject)[0] as
         | HTMLInputElement
         | HTMLTextAreaElement;
     const {value, selectionStart, selectionEnd} = element;
-    const maxLength = element.maxLength === -1 ? Infinity : element.maxLength;
```

Comment:
No more required to do it manually (`document.execCommand('insertText')` behaves like native browser with `attr.maxlength`)